sonar-java-plugin/src/main/resources/org/sonar/l10n/java/rules/java/S6877.html

```diff
@@ -0,0 +1,43 @@
+<h2>Why is this an issue?</h2>
```

Comment:
This content will need to be updated from RSPEC once https://github.com/SonarSource/rspec/pull/3690 is approved and merged.